vignettes/Simulation_PoissonDat.Rmd

```diff
@@ -0,0 +1,291 @@
+---
+title: "Simulation Tutorial (Poisson data)"
+output: 
+  bookdown::html_document2:
+    base_format: rmarkdown::html_vignette
+    fig_caption: yes
+vignette: >
+  %\VignetteIndexEntry{Simulation Tutorial (Poisson data)}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+  
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+In this tutorial, we'll start by generating some simulated spatio-temporal data, with the city of Bristol as our study region. Next, we'll walk you through the steps of running the Bayesian Hierarchical Model (BHM) using the INLA-SPDE approach. Finally, we'll demonstrate the use the key functionality provided by the `fdmr` package.
+
+# Data simulation
+
+First we simulate some Poisson distributed data observed at 55 spatial locations in Bristol, UK and across 6 time points. We use the `load_tutorial_data function` to load in the geographical information for the spatial locations in Bristol.
+
+```{r loadsp}
+sp_data <- fdmr::load_tutorial_data(dataset = "priors", filename = "spatial_dataBris.rds")
+
+```
+
+Then we simulate the Poisson data.
+
+```{r simdat} 

```

Comment:
is it possible to break these out into markdown comments instead of having them as comments in the code? that would make it more readable and also make it more likely to be read. alternatively, we may not care about explaining and drawing the reader ot the data we are building in the simulation.

Reply:
It's resolved now. I moved some comments outside the code chunks, and also deleted the unnecessary ones.